test/Encrypt/BlockCipherTest.php

```diff
@@ -86,11 +86,11 @@ public function testGetSetEncryption(): void
         $filter = new BlockCipherEncryption(['key' => 'testkey']);
         $filter->setVector('1234567890123456');
         $filter->setEncryption(
-            ['algorithm' => 'blowfish']
+            ['algorithm' => 'aes']
         );
         $this->assertSame(
             [
-                'algorithm'     => 'blowfish',
+                'algorithm'     => 'aes',
```

Comment:
Reason for this change: blowfish does not appear to be available on 8.1 with OpenSSL 3.x - possibly "bf" might work, but I got fed up force-pushing random strings to find something that was universally supported. I believe 'aes' is the default, hence the comment.